packages/venia-ui/lib/components/ScrollAnchor/scrollAnchor.css

```diff
@@ -0,0 +1,15 @@
+.anchor {
+    position: absolute;
+    top: calc(var(--venia-global-header-minHeight) * -1px - 5px);
```

Comment:
For some reason, this is not working. Calc and var don't seem to like each other. If I directly using `var(--venia-global-header-minHeight)` it seems to work but since our intention is use a negative offset, it isn't gonna work.